app/frontend/components/pages/exchange/exchange.module.scss

```diff
@@ -0,0 +1,32 @@
+@import '../../../variables.scss';
+
+.content {
+  display: flex;
+  flex: 1;
+  justify-content: center;
+}
+
+.card {
+  box-sizing: content-box;
+  max-width: 568px;
+  width: 100%;
+  height: 398px;
+  @media screen and (max-width: $breakpoint-mobile) {
+    box-sizing: border-box;
+    display: flex;
+    max-width: unset;
+    justify-content: center;
+    height: unset;
+  }
+}
+
+.iframeWrapper {
+  box-sizing: content-box;
+  max-width: 568px;
+  width: 100%;
+  height: 398px;
+  @media screen and (max-width: $breakpoint-mobile) {
+    box-sizing: border-box;
+    height: unset;
+  }
+}
```

Comment:
what's the point of this wrapper since it's just a subset of its parent?

Reply:
Valid point, removed that part of code

Reply:
Please remove the "iframeWrapper" CSS as well

Reply:
Fixed